README.md

```diff
@@ -151,7 +151,7 @@ pytest $MRC_ROOT/python
 ### Docker Installation
 A Dockerfile is provided at `$MRC_ROOT` and can be built with
 ```bash
-docker build -t mrc:latest .
+DOCKER_BUILDKIT=1 docker build -t mrc:latest .
```

Comment:
> BuildKit is the default builder for users on Docker Desktop and Docker Engine v23.0 and later.

Question: Should we communicate that this is only necessary prior to v23.0?

Reply:
I think, since its harmless for users with never versions it isn't needed. Also MRC doesn't have as wide of an audience as Morpheus.